src/components/NewProjectMemberButton.vue

```diff
@@ -112,8 +112,9 @@ const addMember = async () => {
       <Select
         v-model="newMemberRole"
         inputId="role"
-        :options="[...memberRoles]" 
-        optionValue="value"
+         :options="[...memberRoles]"
```

Comment:
Please add a place holder as well.

```suggestion
        placeholder="Select a role"
        :options="[...memberRoles]"
```